### ISSUE TITLE
Bitcore compatibility

### ISSUE THREAD

**Issue by otech47** [lightningnetwork/lnd#891]
Is there any interest or current efforts to allow for compatibility with the [Bitcore](https://bitcore.io/) implementation? Has this been explored and/or deemed unfeasible?

I noticed it jumped to the 2nd most used implementation on [Coindance](https://coin.dance/nodes) and I know that Node is very popular among new and talented developers. Any guidance here from the community would be helpful and I'd like to see if this is feasible to improve compatibility and hopefully developer adoption of Lightning

**Comment by Roasbeef** [comment]
Why would you use `bitcore` of `bitcoind`? As it's built on `bitcoind`.

**Comment by otech47** [comment]
I personally don't have a good answer to that, I'm just assuming that those ~3% of nodes have their reasons and wonderee

**Comment by Roasbeef** [comment]
Gotcha, what I'm saying is that `bitcore` _requires_ `bitcoind`, and we already support `bitcoind`. The `bitcore` element is just a nicer API for developers, and additional indexes FWIW.

**Comment by otech47** [comment]
I personally don't have a good answer to that, I'm just assuming that those ~3% of nodes have their reasons and wondered if it would be worth looking into since btcd is even more uncommon

**Comment by otech47** [comment]
Right that's the premise I guess that developer-friendliness leads to higher adoption

**Comment by Roasbeef** [comment]
In our case, it's just a backend source. So I don't think those attributes really make using their RPC server more attractive.

**Comment by otech47** [comment]
Ok I see. I'm also just coming at this with the goal optimizing for the experience of an average, experienced programmer following the lnd installation instructions. I went through and used btcd just because it was came first sequentially in the docs, but if I happened to have found bitcore easier to run because of my own familiarity with Node, then offering more guidance there would be welcome.

Maybe it's even a simpler implementation technically since it uses bitcoind, but again more than anything it's that process of reading installation docs to getting a lightning node up and running that I'm thinking about

Thanks for the insight!
